### PR TITLE
Device - Add support for "Smartpack R"

### PR DESCRIPTION
**Pull request by mrwold**
Add support for "Smartpack R"

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
Can we maybe just include the string "smartpack", or does that conflict with other Eltek devices that use other OS?

File: includes/definitions/enexus.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ discovery:
         snmpget:
             oid: .1.3.6.1.4.1.12148.10.2.6.0
             op: regex
-            value: '/Micropack System|Smart[Pp]ack S|SmartPack2 Touch|SmartPack2 Syst.|SPS-FPS|Flatpack S|Micropack 1U/'
+            value: '/Micropack System|Smart[Pp]ack S|R|SmartPack2 Touch|SmartPack2 Syst.|SPS-FPS|Flatpack S|Micropack 1U/'
```
Comment (murrant) [inline includes/definitions/enexus.yaml:20]: Also, this regex is wrong, includes anything with an `R` in the snmpget

**Comment by murrant** [comment]
I have a better idea, we could change this to: 
```yaml
discovery:
    -
        sysObjectID:
            - .1.3.6.1.4.1.12148.10
    -
        snmpget:
            oid: .1.3.6.1.4.1.12148.10.2.6.0
            op: '!='
            value: false
```

That way if it responds to that oid with any value, it will work.  Is it true that some of these devices don't have SNMPv2-MIB::sysObjectID.0?

**Comment by mrwold** [comment]
Regarding "SNMPv2-MIB::sysObjectID.0" I do not know, we currently only use Eltek SmartPack R. 
Unfortunately I have a issue generating test-data due to security policy, so I have attached screenshot as proof
image

**Comment by peelman** [comment]
So I applied this, as we have a solitary SmartpackR in our fleet, along with a multitude of SmartpackS systems.  The problem, as I see it, is that .1.3.6.1.4.1.12148.10.2.6.0 is user-configurable.  This isn't the default, but it does complicate trying to use that as something to populate meaningful info into Libre with.

image

`.1.3.6.1.4.1.12148.10.13.8` would be a much more meaningful place to look for OIDs about the control unit.  That's where the current discovery YAML pulls the software version info from.

**Pull request merged**